src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -151,7 +151,7 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
           fullscreen={fullscreen}
           hasBackButton={hasBackButton}
         >
-          <AnnouncementMessage message={venue.bannerMessage} />
+          <AnnouncementMessage banner={venue.banner} announcementForUser />
```

Comment:
What is the distinction between having `announcementForUser` and not? I assume it's used to determine how it renders in admin panel vs normally or similar?

Reply:
`used to determine how it renders in admin panel vs normally` exactly that

Reply:
Fair. That whole piece of this PR felt.. odd. But I guess if it works.. it’s probably fine..